packages/app-sudo/src/index.css

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2017-2019 @polkadot/app-123code authors & contributors
+/* This software may be modified and distributed under the terms
+/* of the Apache-2.0 license. See the LICENSE file for details. */
+
+.sudo--summary {
```

Comment:
Since it is new, probably would have gone for styled-components here rather instead of an additional CSS file.